engine/src/main/java/org/terasology/engine/modes/loadProcesses/PostBeginSystems.java

```diff
@@ -41,7 +47,12 @@ public String getMessage() {
     @Override
     public boolean step() {
         if (componentSystems.hasNext()) {
-            componentSystems.next().postBegin();
+            try {
+                currentSystem = componentSystems.next();
+                currentSystem.postBegin();
+            } catch(Exception e){
```

Comment:
Also as a tiny style issue there you're missing a space or two. If you have the Checkstyle plugin set up in IntelliJ it should highlight any such tweaks needed :-)